identity/src/main/java/com/android/identity/MobileSecurityObjectGenerator.java

```diff
@@ -0,0 +1,272 @@
+package com.android.identity;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import co.nstant.in.cbor.CborBuilder;
+import co.nstant.in.cbor.builder.ArrayBuilder;
+import co.nstant.in.cbor.builder.MapBuilder;
+import co.nstant.in.cbor.model.UnicodeString;
+import java.security.PublicKey;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class MobileSecurityObjectGenerator {
```

Comment:
Want docs on all public fields (classes, methods, etc.). In this case, I'd look at e.g. Device{Request, Response}{Generator, Parser} and use similar wording.